tests/test_codegen.py

```diff
@@ -18,6 +18,9 @@ def schema(cls, version):
                     nonnull text not null,
                     defstr text not null default 'default txt',
                     floaty double default 1.0,
+                    boolt boolean default True,
+                    boolf boolean default False,
```

Comment:
If I'm reading correctly, this will only work if these are capitalized in the Python way. What about `TRUE`, `true`, etc.?

Reply:
yeah, they won't work

Reply:
it was all a hack... eval'ing the sql.   but that's been true from the beginning (ie that bug is old)